### PR TITLE
Flask setup

### PR DESCRIPTION
**Pull request by IdoElk**

### CODE REVIEW

File: lmsweb/models.py [hunk 1]
```diff
@@ -174,6 +172,25 @@ def course_get_administrators(course_id):
...
 
+ALL_MODELS = (User, Course, Lecture)
+
+from flask_admin import Admin
+from flask_admin.contrib.peewee import ModelView
+
+# set optional bootswatch theme
+app.config['FLASK_ADMIN_SWATCH'] = 'cerulean'
+
+admin = Admin(app, name='LMS', template_mode='bootstrap3')
```
Comment (yammesicka) [inline lmsweb/models.py]: template=bootstrap3?

File: lmsweb/models.py [hunk 2]
```diff
@@ -174,6 +172,25 @@ def course_get_administrators(course_id):
...
+    admin.add_view(ModelView(m))
+
+
+def create_tables():
+    with database:
+        database.create_tables(ALL_MODELS)
+
+
 def test():
     database.drop_tables([User, Course, Lecture, UserCourseRelationship, UserLectureRelationship])
```
Comment (yammesicka) [inline lmsweb/models.py]: 99 chars in line

File: lmsweb/models.py [hunk 3]
```diff
@@ -1,7 +1,11 @@
 from peewee import *
+from lmsweb import app
 
-DATABASE_NAME = "database.db"
-database = SqliteDatabase(DATABASE_NAME)
+database = PostgresqlDatabase(database=app.config['DB_NAME'],
```
Comment (yammesicka) [inline lmsweb/models.py]:
```python
db_config = {
    'database': app.config['DB_NAME'],
    'user': app.config['USER'],
    'port': app.config['PORT'],
    'host': app.config['HOST_IP'],
    'password': app.config['PASSWORD'],
}

database = PostgresqlDatabase(**db_config)
```

File: lmsweb/models.py (hunk 1 above)
Comment (IdoElk) [reply lmsweb/models.py]: Since bootstrap 2 is the default we would like to use 3

File: lmsweb/models.py (hunk 2 above)
Comment (IdoElk) [reply lmsweb/models.py]: Correct, although this is part of Sny's PR

**Pull request merged**